src/Steps/Release/MergeUpRelease.php

```diff
@@ -78,16 +80,14 @@ protected function mergeRecursive(
     protected function mergeLibrary(OutputInterface $output, LibraryRelease $releasePlanNode, bool $pushToRemote)
     {
         $library = $releasePlanNode->getLibrary();
-        $releaseBranch = $library->getBranch();
-        $minorBranch = str_replace(LibraryRelease::RELEASE_BRANCH_SUFFIX, '', $releaseBranch);
+        $minorBranch = $library->getBranch();
         $majorBranch = $releasePlanNode->getVersion()->getMajor();
 
-        // Merge up to minor branch, then to major branch
-        $this->mergeUp($output, $releasePlanNode, $releaseBranch, $releaseBranch, $minorBranch, $pushToRemote, false);
-        $this->mergeUp($output, $releasePlanNode, $releaseBranch, $minorBranch, $majorBranch, $pushToRemote, true);
+        // Merge up to major branch
+        $this->mergeUp($output, $releasePlanNode, $minorBranch, $majorBranch, $pushToRemote, true);
```

Comment:
Is the `$isMajorBranch` parameter for `mergeUp()` needed anymore? I don't think it is?

Reply:
Yes, it's passed to ConstraintDestabiliser::destabiliseConstraints() which does use it

Reply:
Right, but that's just always going to be true for mergup, right? So we can remove the parameter from mergeup and just pass in true from there.

Reply:
mergeUp() is currently agnostic if it's merging minor->minor or minor->major.

I know that our current workflow is always minor->major, but they may change in the future so may as well just leave it agnostic.  The recommended change would force it to be minor->major (or major->major) ... however all the language and variable names within mergeUp() are written for it being agnostic

tl;dr just leave it as is